### DISCUSSION TITLE
The only way to get DIC for InferenceData is to compute manually?

### DISCUSSION THREAD

**Discussion by Hellobamboobamboo** [lnccbrown/HSSM#558] [General]
To compare DIC of different models, can I do model. dic(), or because it's inferenceData, the only way is to compute manually?

**Comment by digicosmos86** [comment]
We don't have a `model.dic()` yet, but we will add this soon. Please stay tuned :)

**Comment by Hellobamboobamboo** [comment]
That’s great. Loo and dic, which works better here?

Sincerely,
Jingzhu Chen
(she/her/hers)
***@***.***
917 292 3896

M.S. in Applied Statistics for Social Science Research
NYU Fall 2023 Student Employee Excellence Award Winner
Former piano teacher of Sonoma School District
Former concert pianist of SF Bay Area
https://jingzhuchen.quarto.pub/personal-website/

**Comment by frankmj** [comment]
Loo and WAIC are less biased than DIC so preferable

Michael
